src/main/java/io/vertx/core/http/impl/Http1xServerRequest.java

```diff
@@ -88,6 +88,7 @@ public class Http1xServerRequest extends HttpServerRequestInternal implements io
   // Cache this for performance
   private Charset paramsCharset = StandardCharsets.UTF_8;
   private MultiMap params;
+  private Boolean semicolonIsNormalCharInParams;
```

Comment:
we can use a primitive `boolean` here since `params != null` implies `semicolonIsNormalCharInParams` has been set

Reply:
Done.